### PR TITLE
updating packages version

### PR DESCRIPTION
**Pull request by ac4ch**
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet-js-client/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet-js-client/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,29 +47,31 @@
...
     "typedoc": "^0.25.0",
     "typescript": "^5.0.2"
   },
   "dependencies": {
     "@bsv/sdk": "^1.0.37",
     "@types/rollup-plugin-node-builtins": "^2.1.5",
-    "cross-fetch": "^4.0.0"
+    "cross-fetch": "^4.0.0",
+    "pug": "3.0.3",
+    "pug-code-gen": "3.0.3"
```
Comment (dorzepowski) [inline package.json:75]: What? Why?

Comment (ac4ch) [reply package.json:75]: > Pug through 3.0.2 allows JavaScript code execution if an application accepts untrusted input for the name option of the compileClient, compileFileClient, or compileClientWithDependenciesTracked function. NOTE: these functions are for compiling Pug templates into JavaScript, and there would typically be no reason to allow untrusted callers.

and 
*pug-code-gen (npm) <= 2.0.3 3.0.3*

> Pug through 3.0.2 allows JavaScript code execution if an application accepts untrusted input for the name option of the compileClient, compileFileClient, or compileClientWithDependenciesTracked function. NOTE: these functions are for compiling Pug templates into JavaScript, and there would typically be no reason to allow untrusted callers.

Comment (ac4ch) [reply package.json:75]: https://github.com/bitcoin-sv/spv-wallet-js-client/security/dependabot/15
https://github.com/bitcoin-sv/spv-wallet-js-client/security/dependabot/16

Comment (Nazarii-4chain) [reply package.json]: spv-wallet-js-client does not use `pug` templates

Comment (ac4ch) [reply package.json]: but the security alert was there, maybe not directly and some other lib is using it ?

Comment (Nazarii-4chain) [reply package.json]: `better-docs` uses `pug@2.0.4`. I see two options:
1. Wait until `better-docs` will update their deps
2. Change package to `react-docgen` or `jsdoc`

The last better-docs release was 6 months ago.

Comment (ac4ch) [reply package.json]: so should i remove them ? and dismiss alerts, and we will revisit this once better-docs gets upgraded ?

Comment (ac4ch) [reply package.json]: i removed both of them .. alerts are dismissed in security .. pls have look again at this pr and let me know next steps

**Review by Nazarii-4chain — APPROVED** [review]
LGTM

**Comment by ac4ch** [comment]
@dorzepowski @Nazarii-4chain pls have look at this PR again as i upgraded few more packages per depbot

**Review by Nazarii-4chain — APPROVED** [review]

**Pull request closed without merging**